### ISSUE TITLE
Store and index all Ladybird rights field values

### ISSUE THREAD

**Issue by mikeapp** [yalelibrary/YUL-DC#1576]
**Story**

Only the first rights field from Metadata Cloud Ladybird data is being saved into `rights_statement`, see https://github.com/yalelibrary/yul-dc-management/blob/93d4320916adb23d474758dd5fed6cbe530c5914/app/models/parent_object.rb#L234

Remediation of values already copied is out of scope for this ticket.

Sample item:
The item https://collections-test.library.yale.edu/catalog/17105661 has the following rights in Ladybird, but only Rights 2 is in collections-test:
![Screen Shot 2021-09-09 at 2.45.39 PM.png](https://images.zenhubusercontent.com/5e5d472f410278efd81466ab/7a4e643c-7529-4a7e-a110-4470e60689d2)
(The prod version https://collections.library.yale.edu/catalog/17105661 has been edited/updated to include both.)

**Acceptance**
- [ ] All available `rights` fields from Ladybird/Metadata Cloud should be concatenated with newlines and stored in the field.

**Comment by DraxIndustries79** [comment]
Mike will get real world examples from BRBL

**Comment by K8Sewell** [comment]
PR blocked by drop in code coverage in files unrelated to the PR 
- https://github.com/yalelibrary/yul-dc-management/pull/771
- https://coveralls.io/builds/42800747 - dropped by 13%

**Comment by FCRodriguez7** [comment]
Released management v2.48.13

**Comment by K8Sewell** [comment]
As noted in PR the helper method in Blacklight sanitizes html so will need to adjust that to display them on separate lines.  https://collections-uat.library.yale.edu/catalog/17105661 However, it is now able to display multiple rights statements - for newly created parent objects - a metadata update won't trigger the same result.  

![image.png](https://images.zenhubusercontent.com/5eac63d9b58bd7046578e120/173ab2a6-7760-47fb-8c55-42a10b2ee694)

**Comment by K8Sewell** [comment]
Created follow up ticket - https://app.zenhub.com/workspaces/yul-dc-5e50083561915b11fc9e5850/issues/yalelibrary/yul-dc/1613
